WebViewControl.Avalonia/WebViewControl.Avalonia.csproj

```diff
@@ -12,7 +12,7 @@
     <Product>WebViewControl</Product>
     <Copyright>Copyright ©  2019</Copyright>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>2.75.27</Version>
```

Comment:
Don't need to raise webviews version ... you only touched in the react view

Reply:
i thought we needed both, i'll revert